fe/fe-core/src/main/java/com/starrocks/qe/Coordinator.java

```diff
@@ -2796,7 +2796,6 @@ private void attachInstanceProfileToFragmentProfile() {
         }
     }
 
-<<<<<<< HEAD
```

Comment:
remove It?

Reply:
This is added by the other PR #23198, which leads to fail to compile. I just remove it.